frontend/src/app/cameras/camera/camera.service.ts

```diff
@@ -5,12 +5,13 @@ import {HttpClient} from '@angular/common/http';
 import {map} from 'rxjs/operators';
 
 import CAMERAS from './mock-cameras';
-const CAMERA_URL = '';
+const CAMERA_URL = 'http://10.101.1.23:9090/camera/list';
```

Comment:
would be great if camera url considers ip and port from `app/environment/environment.ts`

Reply:
fixed :)